docs/basic_training/config.pt.md

```diff
@@ -286,18 +285,18 @@ Os seguintes protocolos são suportados:
 
 !!! exercise
 
-     Tente executar o script conforme mostrado abaixo, alterando o arquivo `nextflow.config` para o acima usando `singularity`:
+     Tente executar o script conforme mostrado abaixo, alterando o arquivo `nextflow.config` para ser usado com o Singularity:
 
-     ```bash
-     nextflow executar script7.nf
-     ```
+    ```bash
+    nextflow run script7.nf
+    ```
 
-     !!! observação
+    !!! note
 
-         O Nextflow irá baixar a imagem do contêiner automaticamente. Isso levará alguns segundos, dependendo da velocidade da conexão de rede.
+        O Nextflow irá baixar a imagem do contêiner automaticamente. Isto levará alguns segundos, dependendo da velocidade da sua conexão. 
 
 ### Configurar execução com ambientes Conda
 
-O uso de um ambiente Conda também pode ser fornecido no arquivo de configuração adicionando a seguinte configuração no arquivo `nextflow.config`:
+Ambientes Conda também podem ser fornecidos no arquivo de configuração. Basta adicionar a seguinte configuração no arquivo `nextflow.config`:
 
```

Comment:
I think some example code is missing here

Reply:
Thanks for pointing this out, Iara! For some reason, this is missing in the new layout (but still there in training.seqera.io). The line missing is: process.conda = "/home/ubuntu/miniconda2/envs/nf-tutorial". You can add it here. I will add it to the original version in English now.

Reply:
Fixed in https://github.com/nextflow-io/training/pull/189